### ISSUE TITLE
transcribe from existing audio file

### ISSUE THREAD

**Issue by JLS7311** [Synaptic-Labs-AI/NeuroVox#2]
Unless I'm missing something, I'm not able to transcribe from an existing audio file. Hot key bindings don't kick off any actions and there's no option for upload through the normal microphone or toolbar icons. 

Very excited about this plug-in as it is exactly what I have been looking for.  If this is purely a user error issue, please accept my apologies and any direction is much appreciated.

**Comment by ProfSynapse** [comment]
I'll look into it!

**Comment by ProfSynapse** [comment]
ok try the latest update (0.1.2) - open up an existing recording in your vault so it's the active note. Then command palette ( ctrl+P) and look for the transcribe option with neurovox. Should work!

**Comment by ProfSynapse** [comment]
yes some video formats will work the problem is size. Right now (and this is something I can't control) whisper which does the transcription has a 25mb limit, which is smaller than most videos. I can probably rig something up though that will separate the audio from the video for transcription but will have to think about it.
